Gemfile.lock

```diff
@@ -0,0 +1,28 @@
+GEM
```

Comment:
Yes, because we're specifying the versions of dependencies that we're explicitly pinning.